webrtc/src/track/track_local/track_local_static_sample.rs

```diff
@@ -66,12 +71,13 @@ impl TrackLocalStaticSample {
             if sample.prev_dropped_packets > 0 {
                 packetizer.skip_samples(samples * sample.prev_dropped_packets as u32);
             }
-            /*println!(
```

Comment:
This change is noise, but since you are touching it anyway can you just remove this?